tests/contrib/google_genai/conftest.py

```diff
@@ -3,70 +3,138 @@
 from typing import Iterator
 from unittest.mock import patch as mock_patch
 
+import mock
 import pytest
 
 from ddtrace.contrib.internal.google_genai.patch import patch
 from ddtrace.contrib.internal.google_genai.patch import unpatch
+from ddtrace.llmobs import LLMObs
 from ddtrace.trace import Pin
 from tests.utils import DummyTracer
 from tests.utils import DummyWriter
+from tests.utils import override_global_config
 
 
 @pytest.fixture
-def mock_tracer(genai):
+def ddtrace_global_config():
+    return {}
+
+
+@pytest.fixture
+def mock_tracer(ddtrace_global_config, genai):
     try:
         pin = Pin.get_from(genai)
         mock_tracer = DummyTracer(writer=DummyWriter(trace_flush_enabled=False))
         pin._override(genai, tracer=mock_tracer)
+        pin.tracer.configure()
+        if ddtrace_global_config.get("_llmobs_enabled", False):
+            LLMObs.disable()
+            LLMObs.enable(_tracer=mock_tracer, integrations_enabled=False, agentless_enabled=False)
         yield mock_tracer
     except Exception:
         yield
 
 
 @pytest.fixture
-def genai():
-    patch()
-    from google import genai
+def mock_llmobs_writer():
+    patcher = mock.patch("ddtrace.llmobs._llmobs.LLMObsSpanWriter")
+    try:
+        LLMObsSpanWriterMock = patcher.start()
+        m = mock.MagicMock()
+        LLMObsSpanWriterMock.return_value = m
+        yield m
+    finally:
+        patcher.stop()
```

Comment:
Can you take a look at https://github.com/DataDog/dd-trace-py/blob/main/tests/contrib/crewai/conftest.py#L191-L198? We have a new way of mocking the span writer instead of relying on magic mock